### DISCUSSION TITLE
posts/pawnyable-0202/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#29] [Announcements]
# posts/pawnyable-0202/

Linux Kernel PWN | 040202 Pawnyable之堆溢出

https://blog.wohin.me/posts/pawnyable-0202/

**Comment by C9Y57** [comment]
我想问下怎么关闭/proc/sys/kernel/kptr_restrict保护啊，我不管把echo 0 > /proc/sys/kernel/kptr_restrict写在/init，/etc/inittab里都没有作用，qemu启动后还是会变成2

**Reply by C9Y57** [reply]
噢噢我找到了，在/etc/init.d/S99pawnyable里面，还是对Linux内核启动是怎么样初始化的不太了解

**Reply by C9Y57** [reply]
顺带一提，想要把kptr_restrict给关了方便调试的话，在我环境上会出现仍然无法查看内核地址的情况，刚好博主另一个博客讲了解决方法：https://blog.wohin.me/posts/kernel-mitigation-ctr-related/

**Reply by brant-ruan** [reply]
哈哈，很强👍

**Comment by zayarht3t** [comment]
I want to ask some questions. Can I get your discord?
